CICD.Tools.ValidatorErrorsToMarkdown/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Microsoft.Build.Locator;
+using Microsoft.CodeAnalysis.MSBuild;
+
+using Skyline.DataMiner.CICD.FileSystem;
+
+namespace Skyline.DataMiner.CICD.Tools.ValidatorErrorsToMarkdown
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        public static async Task<int> Main(string[] args)
+        {
+            var fileLocation = new Option<string>(
+                name: "--fileLocation",
```

Comment:
```suggestion
                name: "--inputFilePath",
```
Better to be consistent input - output    instead of fileLocation and resultLocation.  Using 'filePath" also indicates the tool expects the path to the file.